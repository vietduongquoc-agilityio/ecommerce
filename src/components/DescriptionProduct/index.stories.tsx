import { Meta, StoryObj } from "@storybook/react";
import DescriptionProduct from "./";

const meta: Meta<typeof DescriptionProduct> = {
  title: "Components/DescriptionProduct",
  component: DescriptionProduct,
  tags: ["autodocs"],
};

export default meta;

type Story = StoryObj<typeof DescriptionProduct>;

export const Default: Story = {};
