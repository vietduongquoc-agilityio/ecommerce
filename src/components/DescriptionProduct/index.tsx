import Image from "next/image";
import living from "@/assets/Images/Living.jpg";
import { colors, fonts } from "@/themes";

const DescriptionProduct = () => {
  return (
    <section
      style={{
        width: "100%",
        margin: "0 auto",
        padding: "35px 0 61px 0",
        borderTop: "solid 1px #D9D9D9",
        borderBottom: "solid 1px #D9D9D9",
        display: "flex",
        flexDirection: "column",
        alignItems: "center",
      }}
    >
      <div style={{ width: "1025px" }}>
        <h2
          style={{
            fontSize: fonts.size.lg,
            fontWeight: fonts.weight.medium,
            justifyContent: "center",
            display: "flex",
          }}
        >
          Description
        </h2>
        <p
          style={{
            fontWeight: fonts.weight.regular,
            fontSize: fonts.size.xs,
            color: colors.secondary.splight,
            padding: "35px 0",
          }}
        >
          Embodying the raw, wayward spirit of rock ‘n’ roll, the Kilburn
          portable active stereo speaker takes the unmistakable look and sound
          of Marshall, unplugs the chords, and takes the show on the road.
        </p>
        <p
          style={{
            fontWeight: fonts.weight.regular,
            fontSize: fonts.size.xs,
            color: colors.secondary.splight,
          }}
        >
          Weighing in under 7 pounds, the Kilburn is a lightweight piece of
          vintage styled engineering. Setting the bar as one of the loudest
          speakers in its class, the Kilburn is a compact, stout-hearted hero
          with a well-balanced audio which boasts a clear midrange and extended
          highs for a sound that is both articulate and pronounced. The analogue
          knobs allow you to fine tune the controls to your personal preferences
          while the guitar-influenced leather strap enables easy and stylish
          travel.
        </p>
      </div>
      <div style={{ width: "1240px", display: "flex", marginTop: "55px" }}>
        <Image
          src={living}
          alt={"living1"}
          style={{
            width: "605px",
            height: "348px",
            borderRadius: "10px",
            marginRight: "30px",
          }}
        ></Image>
        <Image
          src={living}
          alt={"living2"}
          style={{ width: "605px", height: "348px", borderRadius: "10px" }}
        ></Image>
      </div>
    </section>
  );
};

export default DescriptionProduct;
