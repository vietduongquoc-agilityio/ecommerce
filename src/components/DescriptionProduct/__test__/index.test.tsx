import { render } from "@testing-library/react";
import DescriptionProduct from "../";

describe("DescriptionProduct", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<DescriptionProduct />);
    expect(asFragment()).toMatchSnapshot();
  });
});
