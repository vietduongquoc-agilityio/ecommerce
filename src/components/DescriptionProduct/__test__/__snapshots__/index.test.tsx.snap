// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionProduct should match snapshot 1`] = `
<DocumentFragment>
  <section
    style="width: 100%; margin: 0px auto; padding: 35px 0px 61px 0px; border-top: 1px solid #D9D9D9; border-bottom: 1px solid #D9D9D9; display: flex; flex-direction: column; align-items: center;"
  >
    <div
      style="width: 1025px;"
    >
      <h2
        style="font-size: 24px; font-weight: 500; justify-content: center; display: flex;"
      >
        Description
      </h2>
      <p
        style="font-weight: 400; font-size: 16px; color: rgb(159, 159, 159); padding: 35px 0px;"
      >
        Embodying the raw, wayward spirit of rock ‘n’ roll, the Kilburn portable active stereo speaker takes the unmistakable look and sound of Marshall, unplugs the chords, and takes the show on the road.
      </p>
      <p
        style="font-weight: 400; font-size: 16px; color: rgb(159, 159, 159);"
      >
        Weighing in under 7 pounds, the Kilburn is a lightweight piece of vintage styled engineering. Setting the bar as one of the loudest speakers in its class, the Kilburn is a compact, stout-hearted hero with a well-balanced audio which boasts a clear midrange and extended highs for a sound that is both articulate and pronounced. The analogue knobs allow you to fine tune the controls to your personal preferences while the guitar-influenced leather strap enables easy and stylish travel.
      </p>
    </div>
    <div
      style="width: 1240px; display: flex; margin-top: 55px;"
    >
      <img
        alt="living1"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
        style="color: transparent; width: 605px; height: 348px; border-radius: 10px; margin-right: 30px;"
        width="40"
      />
      <img
        alt="living2"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
        style="color: transparent; width: 605px; height: 348px; border-radius: 10px;"
        width="40"
      />
    </div>
  </section>
</DocumentFragment>
`;
